src/components/common/Calculator.js

```diff
@@ -244,7 +246,13 @@ class Calculator extends Component {
           newHistory,
         );
         break;
+      // APPENDING OPERATIONS
+
       case BUTTON_NAMES.PI: {
+        // start new computation at the end of previous computation
```

Comment:
Why does this happen for pi?

Reply:
@juancarlosfarah Pi acts as a number. For example typing 2,3,4,+,5 gives (234+5) = 239.
Similarly typing pi should output pi and not 239pi

Reply:
Got it. Maybe add that comment.